docs/source/installation.rst

```diff
@@ -11,33 +11,52 @@ Dependencies
 ============
 The general dependencies of VaMPy are
 
+* VTK 8.1.0
 * VMTK 1.4
 * morphMan
 * FEniCS
 * Paramiko
 
-Basic Installation
-==================
 You can choose how to install the dependencies, but the fastest way to get started is to install the dependencies through Anaconda.
-First, install Anaconda or Miniconda (preferably the Python 3.6 version) on your computer.
-Then create two environments, one for `morphMan <https://github.com/KVSlab/morphMan>`_ and `VMTK <http://www.vmtk.org/>`_, and one for `FEniCS <https://fenicsproject.org/>`_ by executing the following in a terminal window::
+You can chose installing either Anaconda or Miniconda on your computer, as described `here <https://conda.io/projects/conda/en/latest/user-guide/install/index.html>`_.
+
+Installing morphMan on Windows or macOS
+=======================================
+The `morphMan <https://github.com/KVSlab/morphMan>`_ software can be installed on Windows and macOS by executing the following in a terminal window::
 
     conda create -n morphman -c vmtk -c morphman morphman python=3.6 paramiko
-    conda create -n fenics -c conda-forge fenics
 
-You can then activate your environment by running ``source activate [ENVIRONMENT NAME]``.
-Windows users may need to install FEniCS as described `here <https://fenicsproject.org/download/>`_.
+You can then activate your environment by running ``source activate morphman``.
+
+Installing morphMan on Linux
+============================
+Linux users may experience various compatibility issues by following the instructions above.
+Therefore, to install the `morphMan <https://github.com/KVSlab/morphMan>`_ software on Linux, execute the following commands::
+
+    conda config --set restore_free_channel true
+    conda create -n morphman -c anaconda vtk=8.1.0
+    conda activate morphman
+    conda install -c vmtk -c morphman morphman paramiko
```

Comment:
@ehsannkh I've tested this on a virtual Linux (Redhat) machine and it resolved the two previous issues we have experienced on Linux. Are you able to test this yourself?

Reply:
@hkjeldsberg Awesome! Nice job! I have also tested on my Ubuntu and it worked. I experienced some small things while going through the instructions which might be worthy to mention.

I got an error complaining about compatibility while installing i.e., "Note that strict channel priority may have removed packages required for satisfiability." because I had channel_priority as strict so I changed it flexible
conda config --set channel_priority flexible

Then, I get the " No module named 'numpy.testing.nosetester' " error while generating mesh, it worked by uninstalling installed numpy and installing pip install numpy==1.16.4.

Reply:
@ehsannkh Awesome! 

I think it's a good idea to mention those minor issues. I'm assuming it happens because Anaconda have already been configured with certain channels which may conflict with the order they are added in here. I can add a sentence about this in the documentation.